src/instructlab/train/linux_train.py

```diff
@@ -256,6 +256,7 @@ def model_generate(user, **kwargs):
             stopping_criteria=stopping_criteria,
             do_sample=True,
             output_logits=True,
+            return_dict_in_generate=True,
```

Comment:
This change is out of scope for my PR, but I'm debugging. Not a permanent change.